src/jquery.mockjax.js

```diff
@@ -667,6 +667,22 @@
 		origSettings.urlParams = paramValues;
 	}
 
+	/**
+	 * Clears handlers that mock given url
+	 * @param url
+	 * @returns {Array}
+	 */
+	function clearByUrl(url) {
+		var results = [];
+		for (var i=0, len=mockHandlers.length; i<len; i++) {
+			var handler = mockHandlers[i];
```

Comment:
I don't think we need another variable here, we could just put this in the `if` statement directly. If you _do_ want to keep it (which is fine) then the variable declaration should be at the top of the function as suggested in my previous comment.